packages/api/src/schema/api-schema.yaml

```diff
@@ -1033,9 +1033,53 @@ components:
       properties:
         name:
           type: string
+          description: Display name
+          example: name
         canPublish:
           type: boolean
           description: Whether a user is allowed to publish audio/video tracks
+          example: true
+    create-room-response:
+      type: object
+      properties:
+        id:
+          type: string
+          description: The ID of the room
+          example: d32ae9e6-c459-4931-9898-e86e2f5e7e16
+    room-user-response:
+      type: object
+      properties:
+        id:
+          type: string
+          description: The ID of the user
+          example: d32ae9e6-c459-4931-9898-e86e2f5e7e16
+        joinUrl:
+          type: string
+          description: Joining URL
+          example: https://meet.livepeer.chat
+        token:
+          type: string
+          description: Joining JWT
+          example: token
+    get-room-user-response:
+      type: object
+      properties:
+        id:
+          type: string
+          description: The ID of the user
+          example: d32ae9e6-c459-4931-9898-e86e2f5e7e16
+        joinedAt:
+          type: integer
+          description: The time the user joined
+          example: 1687771565
+        name:
+          type: string
+          description: The display name of the user
+          example: name
+        isPublisher:
+          type: boolean
+          description: Whether a user is allowed to publish audio/video tracks
+          example: true
```

Comment:
Can you add a `default` here too? I assume it is `true`? Default also serves as example